### PR TITLE
[SC-12454] Support for additional container args

### PR DESCRIPTION
**Pull request by dvdmelamed**

### CODE REVIEW

File: outfile.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+{"statusCode": 200, "headers": {"Content-Type": "application/json", "Access-Control-Allow-Origin": "*", "Access-Control-Allow-Credentials": true}, "body": "{}"}
```
Comment (jit-shlomi) [inline outfile.txt:1]: what's that?

File: action.yml [hunk 2]
```diff
@@ -58,5 +62,5 @@ runs:
         docker pull ${{ inputs.security_control }}
         JSON_GITHUB_EVENT='${{ toJSON(github) }}'
         B64_ENCODED_GITHUB_EVENT=$(echo $JSON_GITHUB_EVENT | base64)
-        docker run --rm -e CONFIG_FILE_PATH=/.jit/jit.yml -e GITHUB_EVENT="$B64_ENCODED_GITHUB_EVENT" -e SOURCE_CODE_FOLDER=/code -e SECURITY_CONTROL_OUTPUT_FILE=${{ inputs.security_control_output_file }} -v $(pwd)/.jit:/.jit -e FAIL_ON_FINDINGS=${{ inputs.fail_on_findings }} -v $(pwd)/code:/code ${{ inputs.security_control }} -- ${{ inputs.security_control_args }}
+        docker run --rm -e CONFIG_FILE_PATH=/.jit/jit.yml ${{ inputs.container_args }} -e GITHUB_EVENT="$B64_ENCODED_GITHUB_EVENT" -e SOURCE_CODE_FOLDER=/code -e SECURITY_CONTROL_OUTPUT_FILE=${{ inputs.security_control_output_file }} -v $(pwd)/.jit:/.jit -e FAIL_ON_FINDINGS=${{ inputs.fail_on_findings }} -v $(pwd)/code:/code ${{ inputs.security_control }} -- ${{ inputs.security_control_args }}
```
Comment (jit-shlomi) [inline action.yml:65]: Thinking about the near future, secrets will probably be a part of the payload and not an env variable.
with that, will we really need this field?

**Review by jit-shlomi — COMMENTED** [review]
cr

File: action.yml (hunk 2 above)
Comment (dvdmelamed) [reply action.yml:65]: i think so, there are other possible args we want to pass, i.e. mounts, permissions...

Comment (jit-shlomi) [reply action.yml:65]: ok

**Review by jit-shlomi — APPROVED** [review]

**Pull request merged**